### PR TITLE
Custom eventshub to support kafka consumer

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

Add a new custom eventshub that can perform all the eventshub functionalities and can also be used as kafka consumer 

## Proposed Changes

- :gift: Add a new eventshub that can replace our existing kafka test images, including options and asserts

### CODE REVIEW

**Comment by slinkydeveloper** [comment]
Depends on https://github.com/knative-sandbox/reconciler-test/pull/225

**Comment by slinkydeveloper** [comment]
Depends on https://github.com/knative-sandbox/reconciler-test/pull/225

File: test/test_images/eventshub/kafka_consumer.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+		s := atomic.AddUint64(&h.seq, 1)
+
+		eventInfo := eventshub.EventInfo{
+			Error:    eventErrStr,
+			Event:    event,
+			Observer: "kafka-receiver",
+			Time:     time.Now(),
+			Sequence: s,
+			Kind:     eventshub.EventReceived,
+			// TODO add additional info with partition, offset and topic
```
Comment (pierDipi) [inline test/test_images/eventshub/kafka_consumer.go:124]: Yeah, good idea!

**Review by pierDipi — COMMENTED** [review]
LGTM

**Comment by matzew** [comment]
@slinkydeveloper is it possible to get a rebased version of this PR?

Thanks :pray:

**Comment by matzew** [comment]
@pierDipi what should we do with this?

**Comment by pierDipi** [comment]
We should keep/rebase and see if something is missing.

**Comment by pierDipi** [comment]
A rebased version https://github.com/knative-sandbox/eventing-kafka-broker/pull/1282.
Thanks, Francesco!

**Pull request closed without merging**